src/pages/Shop/components/AverageRating.tsx

```diff
@@ -0,0 +1,33 @@
+import { useParams } from 'react-router-dom';
+import { useShopTotalReview } from '../hooks/useGetShopReview';
+import Rating from './Rating';
+import StarList from './StarList';
+
+export default function AverageRating() {
+  const { shopId } = useParams();
+  const { data: useShopTotalReviews } = useShopTotalReview(shopId!);
+
+  const average = useShopTotalReviews.statistics.average_rating;
+  const ratings = useShopTotalReviews.statistics.ratings;
+  const totalReviewCount = useShopTotalReviews.total_count;
+  const rateList: ['5', '4', '3', '2', '1'] = ['5', '4', '3', '2', '1'];
```

Comment:
요거는 상수로 빼려고 타입을 따로 정의한 것 같네요. 
```suggestion
  const rateList = ['5', '4', '3', '2', '1'] as const;
```
이렇게 상수로 타입 정의를 해도 되고, 아니면 enum을 사용해서 타입 정의를 해도 될 거 같아요.
(처음에 왜 String 타입으로 작성했나 했는데, ratings 객체의 키값이 String 타입이군요...)